great_expectations/rule_based_profiler/domain_builder/multi_column_domain_builder.py

```diff
@@ -26,6 +26,16 @@ class MultiColumnDomainBuilder(ColumnDomainBuilder):
     This DomainBuilder uses relative tolerance of specified map metric to identify domains.
     """
 
+    exclude_field_names: Set[str] = ColumnDomainBuilder.exclude_field_names | {
```

Comment:
Same note

Reply:
@cdkini Same cause -- it is all rooted in the way `IDDict` behaves.  We may have a JIRA for fixing it.

Reply:
sure thing